compiler/luci/import/include/luci/Import/GraphBuilder.h

```diff
@@ -33,6 +33,12 @@ class GraphBuilder : public GraphBuilderBase
 public:
   virtual ~GraphBuilder() = default;
 
+  // common validate to check number of inputs and single output
```

Comment:
Maybe :D ?
```suggestion
  // common validation to check number of inputs and single output
```

Reply:
Ah, `validate` is the method `validate`... I'll revise this...